### ISSUE TITLE
Docs for FSTree.fromEntries is incorrect

### ISSUE THREAD

**Issue by zackthehuman** [stefanpenner/fs-tree-diff#24]
The usage is:

```
var entries = walkSync.entries(somePath);
var fsTree = FSTree.fromEntries(entries);
```

The docs indicate that you should be passing in an `object` with an `entries` field, but you should really just be passing in the `entries` array directly.

I can clean up the docs -- this issue is just for tracking purposes.

**Comment by stefanpenner** [comment]
@zackthehuman you are correct, good catch.

**Comment by hjdivad** [comment]
@zackthehuman if you have a minute to clean up the docs that would be great; otherwise i can do it

**Comment by zackthehuman** [comment]
@hjdivad I thought I created a PR but I think I accidentally a thing. Hold on.

**Comment by zackthehuman** [comment]
@hjdivad See PR https://github.com/stefanpenner/fs-tree-diff/pull/25

**Comment by hjdivad** [comment]
Fixed by #25
